examples/test_window.rs

```diff
@@ -6,17 +6,43 @@ use {
 };
 
 struct Stage {
-    egui_ctx: std::sync::Arc<egui::Context>,
+    egui_ctx: egui::CtxRef,
     raw_input: egui::RawInput,
     start_time: Instant,
     painter: Painter,
 }
 
+impl Stage {
+    fn new(ctx: &mut mq::Context) -> Self {
+        let egui_ctx = egui::CtxRef::default();
+
+        let pixels_per_point = ctx.dpi_scale();
+        let (width, height) = ctx.screen_size();
+        let screen_size = vec2(width as f32, height as f32) / pixels_per_point;
+
+        let raw_input = egui::RawInput {
+            screen_rect: Some(egui::Rect::from_min_size(Default::default(), screen_size)),
+            pixels_per_point: Some(pixels_per_point),
+            ..Default::default()
+        };
+
+        Self {
+            egui_ctx,
+            painter: Painter::new(ctx),
+            raw_input,
+            start_time: Instant::now(),
```

Comment:
Note that `Instant` do not really work on wasm. 
Alternatives are:
`miniquad::now()` - will give time as f64
or https://github.com/sebcrozet/instant - uses `miniquad::now` under the hood and will give a struct similar to time::Instant